### PR TITLE
[Playwright] PEPPER-1118 New test for research kits prefix check

### PR DESCRIPTION
**Pull request by aweng98**
[PEPPER-1118](https://broadworkbench.atlassian.net/browse/PEPPER-1118)

Prefix check for blood kit with either a Canada or New York address. Test for saliva kit will be in a separate test.

Also, fixing broken LMS tests due to failure in screenshots comparison. Most recent CircleCI [run](https://app.circleci.com/pipelines/github/broadinstitute/ddp-angular/25824/workflows/087b9996-9e2e-49e7-b969-decdb8d06317/jobs/78939/tests).

[PEPPER-1118]: https://broadworkbench.atlassian.net/browse/PEPPER-1118?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: playwright-e2e/dss/component/table.ts
Comment (aweng98) [inline playwright-e2e/dss/component/table.ts:334]: Generic version of function [`getParticipantDataAt`](https://github.com/broadinstitute/ddp-angular/blob/7b41d2257327ae35105a6064658aafa31ddbd4af/playwright-e2e/dsm/component/tables/participant-list-table.ts#L46)

File: playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-prefix-check.spec.ts
Comment (aweng98) [inline playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-prefix-check.spec.ts:44]: Limit test for one study as we don't have limited amount of participants. One participant with too many kits can encounter error.

File: playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-prefix-check.spec.ts
Comment (Rinchan20) [inline playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-prefix-check.spec.ts:172]: nice! (handling of timing for label creation, I mean)

File: playwright-e2e/utils/test-utils.ts
Comment (Rinchan20) [inline playwright-e2e/utils/test-utils.ts:256]: good comment to add (thumbs up - written since I can't find emoji for this textbox)

**Review by Rinchan20 — APPROVED** [review]

**Comment by aweng98** [comment]
@Rinchan20 Hi, I have added more changes to this PR after your approval. All tests are now passing again on CircleCI. I'm going to merge right now. Let me know if you would like to see more changes after merge. Thank you.

**Pull request merged**
